dynamo/tools/recipes.py

```diff
@@ -89,6 +89,9 @@ def recipe_kin_data(
     from ..preprocessing import recipe_monocle
     from ..preprocessing.utils import pca, detect_datatype
 
+    keep_filtered_cells = DynamoConfig.check_config_var(keep_filtered_cells, DynamoConfig.KEEP_FITLERED_CELLS_KEY)
+    keep_filtered_genes = DynamoConfig.check_config_var(keep_filtered_genes, DynamoConfig.KEEP_FITLERED_GENES_KEY)
+
```

Comment:
what are you doing here? as discussed in the above, keep the good default for those analysis receipes --- that is, creating new recipe arguments for them. For the preprocessing recipe, recipe_monocle, use the default values

Reply:
These are written in the "default" mode DynamoConfig.